packages/backend/src/open_payments/grant/service.ts

```diff
@@ -34,24 +36,30 @@ export interface GrantOptions {
   accessActions: AccessAction[]
 }
 
-export interface CreateOptions extends GrantOptions {
-  accessToken?: string
+export interface UpdateOptions {
+  accessToken: string
+  managementUrl: string
   expiresIn?: number
 }
```

Comment:
```suggestion
export type UpdateOptions = Omit<AccessToken, 'access'>
```
Import `AccessToken` from `open-payments`.
(This would have `expires_in` instead of `expiresIn`).
Maybe we could also update `CreateOptions` to be based on `open-payments` types.

Reply:
and `manage` instead of `managementUrl`
and `value` instead of `accessToken`
:thinking: 
Nevermind